### ISSUE TITLE
`anemic-getter` lint doesn't detect and complain about redundant renaming

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#1110]
Consider this code:

```
[title] > book
  title > t
```

Here, we introduce a getter `t`. This is redundant, since all it does is giving access to `title`.

Let's create a lint to catch this and complain.

**Comment by yegor256** [comment]
@rultor release, tag is `0.4.0`
